src/diffusers/pipelines/audio_diffusion/pipeline_latent_audio_diffusion.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Union
+
+from ...models import AutoencoderKL, Mel, UNet2DConditionModel
+from ...schedulers import DDIMScheduler, DDPMScheduler
+from .pipeline_audio_diffusion import AudioDiffusionPipeline
+
+
+class LatentAudioDiffusionPipeline(AudioDiffusionPipeline):
```

Comment:
We don't want to inherit from other pipelines to keep the dependencies as small as possible. What exactly do we need this pipeline for in the first place?

Reply:
Done.